<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MAME Plus! GUI 0.96a</string>
  </property>
  <property name="dockNestingEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>47</y>
     <width>104</width>
     <height>337</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <item row="0" column="0" >
     <layout class="QVBoxLayout" name="layMainView" >
      <property name="topMargin" >
       <number>2</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionExitStop" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionPicture_Area" />
    <addaction name="actionVerticalTabs" />
    <addaction name="separator" />
    <addaction name="actionLargeIcons" />
    <addaction name="actionDetails" />
    <addaction name="actionGrouped" />
    <addaction name="separator" />
    <addaction name="actionRefresh" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>Options</string>
    </property>
    <addaction name="actionDefaultOptions" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuView" />
   <addaction name="menuOptions" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>384</y>
     <width>624</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>21</y>
     <width>162</width>
     <height>26</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Toolbar</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionPicture_Area" />
   <addaction name="separator" />
   <addaction name="actionLargeIcons" />
   <addaction name="actionDetails" />
   <addaction name="actionGrouped" />
   <addaction name="separator" />
   <addaction name="actionDefaultOptions" />
   <addaction name="actionAbout" />
  </widget>
  <widget class="QToolBar" name="toolBarSearch" >
   <property name="geometry" >
    <rect>
     <x>162</x>
     <y>21</y>
     <width>462</width>
     <height>26</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Search</string>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2" >
   <property name="geometry" >
    <rect>
     <x>368</x>
     <y>47</y>
     <width>256</width>
     <height>337</height>
    </rect>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle" >
    <string>GUI Log</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>22</y>
      <width>256</width>
      <height>315</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <item row="0" column="0" >
      <widget class="QTextBrowser" name="textBrowserFrontendLog" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_7" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>47</y>
     <width>256</width>
     <height>337</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle" >
    <string>Folder List</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_14" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>22</y>
      <width>256</width>
      <height>315</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <item row="0" column="0" >
      <widget class="QTreeWidget" name="treeFolders" >
       <property name="headerHidden" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionDefaultOptions" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="mamepguimain.qrc" >
     <normaloff>:/res/32x32/applications-system.png</normaloff>:/res/32x32/applications-system.png</iconset>
   </property>
   <property name="text" >
    <string>Default Options</string>
   </property>
  </action>
  <action name="actionRefresh" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="mamepguimain.qrc" >
     <normaloff>:/res/32x32/view-refresh.png</normaloff>:/res/32x32/view-refresh.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionExitStop" >
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="mamepguimain.qrc" >
     <normaloff>:/res/32x32/help-browser.png</normaloff>:/res/32x32/help-browser.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionDetails" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="mamepguimain.qrc" >
     <normaloff>:/res/mame32-view-detail.png</normaloff>:/res/mame32-view-detail.png</iconset>
   </property>
   <property name="text" >
    <string>Details</string>
   </property>
  </action>
  <action name="actionGrouped" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="mamepguimain.qrc" >
     <normaloff>:/res/mame32-view-group.png</normaloff>:/res/mame32-view-group.png</iconset>
   </property>
   <property name="text" >
    <string>Grouped</string>
   </property>
  </action>
  <action name="actionLargeIcons" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="mamepguimain.qrc" >
     <normaloff>:/res/mame32-view-licon.png</normaloff>:/res/mame32-view-licon.png</iconset>
   </property>
   <property name="text" >
    <string>Large Icons</string>
   </property>
  </action>
  <action name="actionPicture_Area" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="mamepguimain.qrc" >
     <normaloff>:/res/mame32-show-snap.png</normaloff>:/res/mame32-show-snap.png</iconset>
   </property>
   <property name="text" >
    <string>Picture Area</string>
   </property>
   <property name="shortcut" >
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionVerticalTabs" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Vertical Tabs</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mamepguimain.qrc" />
 </resources>
 <connections/>
</ui>
